feat: Configure disable subdomain isolation (#2548)
codenvy/che-docs: modules/administration-guide/partials/proc_applying-the-github-oauth-app-secret.adoc

    app.kubernetes.io/component: oauth-scm-configuration
  annotations:
    che.eclipse.org/oauth-scm-server: github
    che.eclipse.org/scm-server-endpoint: __<github_server_url>__ <2>
    che.eclipse.org/scm-github-disable-subdomain-isolation: "__<true_or_false>__" <3>
type: Opaque
data:
  id: __<Base64_GitHub_OAuth_Client_ID>__ <4>
  secret: __<Base64_GitHub_OAuth_Client_Secret>__ <5>
----
<1> The {prod-short} namespace. The default is `{prod-namespace}`.

> <2> A *GitHub Enterprise Server URL*. By default `https://github.com` is used for the `SAAS` version.

Reviewer: ```suggestion
<2> A *GitHub Enterprise Server URL*. By default, `https://github.com` is used for the `SAAS` version.
```

  annotations:
    che.eclipse.org/oauth-scm-server: github

>     che.eclipse.org/scm-server-endpoint: <github_server_url> <2>

Reviewer: ```suggestion
    che.eclipse.org/scm-server-endpoint: __<github_server_url>__ <2>
```

>     che.eclipse.org/scm-github-disable-subdomain-isolation: <disable_subdomain_isolation> <3>

Reviewer: ```suggestion
    che.eclipse.org/scm-github-disable-subdomain-isolation: "__<true_or_false>__" <3>
```

type: Opaque
data:
  id: __<Base64_GitHub_OAuth_Client_ID>__ <4>
  secret: __<Base64_GitHub_OAuth_Client_Secret>__ <5>
----
<1> The {prod-short} namespace. The default is `{prod-namespace}`.
<2> A *GitHub Enterprise Server URL*. By default `https://github.com` is used for the `SAAS` version.

> <3> Disables subdomain isolation, possible values are `"true"` or `"false"`

Reviewer: ```suggestion
<3> Subdomain isolation for GitHub Enterprise Server. To disable subdomain isolation, set this to `"true"`. To enable subdomain isolation, set this to `"false"`.
```
